packages/usuario/src/drivers/mysql/models/usuario.js

```diff
@@ -0,0 +1,87 @@
+const { Model, DataTypes, Sequelize } = require('sequelize');
+
+const { PERSONAS_TABLE } = require('./personas.models');
+const { ROLES_TABLE } = require('./roles.models');
+
+const USUARIOS_TABLE = 'usuario';
+
+const usuariosSchema = {
+	id: {
+		allowNull: false,
+		autoIncrement: true,
+		primaryKey: true,
+		type: DataTypes.INTEGER,
+	},
+	personaId: {
+		allowNull: false,
+		type: DataTypes.INTEGER,
+		field: 'persona_id',
+		references: {
+			model: PERSONAS_TABLE,
+			key: 'id',
+		},
+		/* onUpdate: 'CASCADE',
+		onDelete: 'SET NULL', */
+	},
+	rolId: {
+		allowNull: false,
+		type: DataTypes.INTEGER,
+		field: 'rol_id',
+		references: {
+			model: ROLES_TABLE,
+			key: 'id',
+		},
+		onUpdate: 'CASCADE',
+		onDelete: 'SET NULL',
```

Comment:
I think we should set to null in this field, if we are aiming to create a logical delete. we are going to lose information.